dotnet/src/SemanticKernel.Abstractions/Orchestration/IKernelExecutionContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Orchestration;
+
+/// <summary>
+/// Kernel execution context
+/// </summary>
+public interface IKernelExecutionContext
```

Comment:
Can we name it so that it's clear that it's not a context and that it's purpose to execute functions? E.g. ISKFunctionRunner/Executor/Invoker?

Reply:
REnamed to `IFunctionExecutor`